### PR TITLE
Refs #10086: Exit loop if a particular server returns successfully.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: templates/install_rvm.sh.erb [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
     hkp://pgp.mit.edu
+    hkp://keys.gnupg.net
 )
 
 for server in "${SERVERS[@]}"
 do
-    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
+    success=`su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER`
+
+    if ($success); then
```
Comment (ekohl) [inline templates/install_rvm.sh.erb]: are the braces needed here?

**Comment by ekohl** [comment]
Minor nitpick, but :+1:

**Comment by ehelms** [comment]
@ekohl thanks for the review, updated
@stbenjam or @jlsherrill  either of you good with this slight update to add to the installer PR?

File: templates/install_rvm.sh.erb [hunk 2]
```diff
@@ -3,12 +3,16 @@
...
 )
 
 for server in "${SERVERS[@]}"
 do
-    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
+    success=`su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER`
+
+    if $success; then
+        break
+    fi
```
Comment (jlsherrill) [inline templates/install_rvm.sh.erb]: would it make more sense to check the return value of the command?  ($?)

Testing locally i still hit break even if success is empty (implying no output)

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**